NDTensors/ext/NDTensorsAMDGPUExt/NDTensorsAMDGPUExt.jl

```diff
@@ -0,0 +1,19 @@
+module NDTensorsAMDGPUExt
+
+using NDTensors
+using Adapt
+using Functors
+using AMDGPU
+using AMDGPU.Runtime.Mem
+using AMDGPU.rocBLAS
+using AMDGPU.rocSOLVER
```

Comment:
Let's explicitly list the functions we are using from each module, i.e. `using Adapt: adapt, ...`.

Reply:
Okay, I have done this and moved all the usings to appropriate folders